### PR TITLE
added access to the statements explaining cluster role 

### PR DESCRIPTION
**Pull request by dmarinere**
I added access to the statement explaining cluster role  permissions in this document to make it clearer.

### CODE REVIEW

File: content/en/docs/reference/access-authn-authz/rbac.md [hunk 1]
```diff
@@ -54,8 +54,8 @@ it can't be both.
 
 ClusterRoles have several uses. You can use a ClusterRole to:
 
-1. define permissions on namespaced resources and be granted within individual namespace(s)
-1. define permissions on namespaced resources and be granted across all namespaces
+1. define permissions on namespaced resources and be granted access within individual namespace(s)
+1. define permissions on namespaced resources and be granted access across all namespaces
```
Comment (reylejano) [inline content/en/docs/reference/access-authn-authz/rbac.md:58]: ClusterRoles are a set of permissions. ClusterRoles do not grant access. RoleBindngs and ClusterRoleBindings grant the permissions set by a ClusterRole to a user or set of users

File: content/en/docs/reference/access-authn-authz/rbac.md [hunk 2]
```diff
@@ -54,8 +54,8 @@ it can't be both.
 
 ClusterRoles have several uses. You can use a ClusterRole to:
 
-1. define permissions on namespaced resources and be granted within individual namespace(s)
-1. define permissions on namespaced resources and be granted across all namespaces
+1. define permissions on namespaced resources and be granted access within individual namespace(s)
```
Comment (kbhawkey) [inline content/en/docs/reference/access-authn-authz/rbac.md:57]: I agree that the statements are not entirely clear.
Ping /sig auth
Thanks!

**Comment by kbhawkey** [comment]
Preview:
https://deploy-preview-34716--kubernetes-io-main-staging.netlify.app/docs/reference/access-authn-authz/rbac/#role-and-clusterrole

**Comment by kbhawkey** [comment]
Thanks for contributing @dmarinere .
/approve

**Pull request merged**